### PR TITLE
fix: revert use named return changes

### PR DESCRIPTION
**Pull request by ahramy**
[AXE-6230](https://axelarnetwork.atlassian.net/browse/AXE-6230)

[AXE-6230]: https://axelarnetwork.atlassian.net/browse/AXE-6230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axelar-network/axelar-gmp-sdk-solidity",
-  "version": "6.0.0",
+  "version": "6.0.1",
```
Comment (milapsheth) [inline package.json:3]: don't need to change this anymore. new release process is active now. update npm deps locally, run `npm run cs` to create a patch release changelog and push the changeset file to this PR. Then trigger the `Create Release PR` action to trigger the patch release

**Review by milapsheth — APPROVED** [review]

File: .changeset/fair-trains-own.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@axelar-network/axelar-gmp-sdk-solidity': patch
+---
+
+fit: revert use named return changes
```
Comment (milapsheth) [inline .changeset/fair-trains-own.md:5]:
```suggestion
reverted use of named return value in RolesBase to not change bytecode of InterchainToken
```

File: package.json (hunk 1 above)
Comment (milapsheth) [inline package.json:3]: revert this version bump. the release PR will handle it

Comment (ahramy) [reply package.json]: got it. used `npm run cs` command for the changeset.

Comment (ahramy) [reply package.json]: got it. reverted version bump

File: .changeset/fair-trains-own.md (hunk 2 above)
Comment (ahramy) [reply .changeset/fair-trains-own.md]: thank you. updated.

**Pull request merged**